test/jdk/jdk/nullCaller/exeNullCallerTest.cpp

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+#include "CallHelper.hpp"
+
+/* Test for JDK-8280902
+ * ResourceBundle::getBundle may throw NPE when invoked by JNI code with no caller frame
+ */
+void JDK_8280902(JNIEnv* env) {
+    StaticCall m_ResourceBundle_getBundle { env,
+        "java/util/ResourceBundle", "getBundle",
+        "(Ljava/lang/String;)Ljava/util/ResourceBundle;" };
+    InstanceCall m_ResourceBundle_getString { env,
+        "java/util/ResourceBundle", "getString",
+        "(Ljava/lang/String;)Ljava/lang/String;" };
+    StaticCall m_ResourceBundle_clearCache { env,
+        "java/util/ResourceBundle", "clearCache", "()V"
+    };
+
+    // The following should not throw
+    // b = ResourceBundle.getBundle("NullCallerResource");
+    jobject b =m_ResourceBundle_getBundle.callReturnNotNull(
+        env->NewStringUTF("open/NullCallerResource") );
+
+    // msg = b.getString("message");
+    jstring msg = (jstring) m_ResourceBundle_getString.callReturnNotNull(b, env->NewStringUTF("message"));
+
+    // check the message
+    if (std::string("Hello!") != env->GetStringUTFChars(msg,NULL)) {
```

Comment:
nit: a space after the comma `(msg, NULL)` consistent with the format in this local file.